service/worker/esanalyzer/workflow.go

```diff
@@ -206,6 +213,9 @@ func (w *Workflow) emitWorkflowVersionMetrics(ctx context.Context) error {
 				return err
 			}
 			for _, workflowType := range domainWorkflowVersionCount.WorkflowTypes {
+				w.analyzer.tallyScope.Tagged(
```

Comment:
nit: maybe have` {domainTag: domainName, workflowTypeTag: workflowType.AggregateKey} ` wrapped into var to dedup with loop below.

Reply:
Thanks! This is removed and emitted in a new workflow